### PR TITLE
Fix broken hyperlink

### PR DESCRIPTION
**Pull request by droctothorpe**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
This tiny PR just fixes a broken hyperlink in the FAQ.

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: cluster-autoscaler/README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Cluster Autoscaler is a tool that automatically adjusts the size of the Kubernet
 
 # FAQ/Documentation
 
-An FAQ is available [HERE](./FAQ.md).
+A FAQ is available [HERE](./FAQ.md).
```
Comment (Shubham82) [inline cluster-autoscaler/README.md:13]: IMO, `An FAQ`  is grammatically accurate.

File: cluster-autoscaler/FAQ.md [hunk 2]
```diff
@@ -862,7 +862,7 @@ Events:
 ```
 
 This limitation was solved with
-[volume topological scheduling](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/storage/volume-topology-scheduling.md)
+[volume topological scheduling](https://github.com/kubernetes/design-proposals-archive/blob/acc25e14ca83dfda4f66d8cb1f1b491f26e78ffe/storage/volume-topology-scheduling.md)
```
Comment (Shubham82) [inline cluster-autoscaler/FAQ.md:865]:
```suggestion
[volume topological scheduling](https://github.com/kubernetes/design-proposals-archive/blob/main/storage/volume-topology-scheduling.md)
```

**Review by Shubham82 — CHANGES_REQUESTED** [review]

File: cluster-autoscaler/FAQ.md (hunk 2 above)
Comment (Shubham82) [reply cluster-autoscaler/FAQ.md:865]: Please change the branch name to main.

File: cluster-autoscaler/README.md (hunk 1 above)
Comment (lzhecheng) [reply cluster-autoscaler/README.md:13]: Agree

Comment (droctothorpe) [inline cluster-autoscaler/README.md:13]:
```suggestion
An FAQ is available [HERE](./FAQ.md).
```

Comment (droctothorpe) [reply cluster-autoscaler/README.md]: Duly noted. Removed from the PR. Thanks!

**Comment by droctothorpe** [comment]
Thanks for the input. Merged suggestions. Squashed. Kept you as co-author, @Shubham82. Updated commit message, PR title, and PR description accordingly.

What's the joke?
1 line PR: 10 comments.
10,000 line PR: 0 comments. 

🙃

**Comment by Shubham82** [comment]
Thanks, @droctothorpe 
it looks good to me.
/lgtm

**Comment by droctothorpe** [comment]
What's the next step, @Shubham82?

**Comment by Shubham82** [comment]
@droctothorpe you can ask [feiskyer](https://github.com/feiskyer) for approval by assiging the issue to him.
See this [comment](https://github.com/kubernetes/autoscaler/pull/5987#issuecomment-1652249884)

**Review by mwielgus — APPROVED** [review]
/lgtm
/approve

**Comment by droctothorpe** [comment]
Thank you, frondz! Does this mean I get to go to the contributor lounge at kubecon? 😂 Hopefully more substantive contributions to follow.

**Pull request merged**